webcompat/static/css/src/issue-wizard.css

```diff
@@ -496,6 +496,7 @@
   font-weight: 200;
   margin-bottom: 17px;
   padding: 5px 32px 7px 8px;
+  transition: box-shadow .3s;
```

Comment:
This transition on `.low .text-field` is not needed I think, as long as it's specified in `.issue-form .text-field`